Strata/Languages/Dyn/Examples/Arithmetic.lean

```diff
@@ -0,0 +1,26 @@
+/-
+  Copyright Strata Contributors
+
+  SPDX-License-Identifier: Apache-2.0 OR MIT
+-/
+
+import Strata.Languages.Dyn.Dyn
+
+namespace Strata
+namespace Dyn
+
+def ArithmeticPgm :=
+#strata
+program Dyn;
+
+def test_arithmetic (dummy : Any) -> Any
+{
+  var sum : Any;
+  sum = int_to_Any(1) + int_to_Any(2);
+  return sum;
+}
+
+#end
```

Comment:
On one hand, the syntax of Dyn can be whatever we want. On the other hand, I think it would be easier to approach (and require less decision making on our part) to, by default, make it consistent with either TypeScript or Python. Here, we have TypeScript-like curly braces and `var` declarations but Python `def` declarations and even C-like C `alloc` and `free`.

Since TypeScript is our first target, I'd lean towards using TypeScript syntax. We can stick a nicer core subset of TypeScript (e.g., only arrow functions), for example,

```typescript
const test_arithmetic: any => any = (dummy) => {
  var sum: any;
  sum = 1 + 2;
  return sum;
}
```